scripts/set-alias-page.py

```diff
@@ -225,6 +233,14 @@ def main():
         default="",
         help='page name in the format "platform/alias_command.md"',
     )
+    parser.add_argument(
+        "-l",
+        "--language",
+        type=str,
+        required=False,
+        default="",
+        help='language in the format "fr" or "pt_BR"',
```

Comment:
```suggestion
        help='language in the format "ll" or "ll_CC" as in "fr" or "pt_BR"',
```
I know we should keep the help short, but saying examples are formats doesn't seem right to me. I think this is still short enough, what do you think?